### PR TITLE
3026: Add name and to profile staff

### PR DESCRIPTION
**Pull request by KristianB14**
https://platform.dandigbib.org/issues/3026

### CODE REVIEW

File: themes/ddbasic/sass/components/user.scss [hunk 1]
```diff
@@ -36,6 +36,7 @@
     }
     h1 {
       @include font('display');
+      float: left;
```
Comment (kasperg) [inline themes/ddbasic/sass/components/user.scss]: Is it intentional that this float is added?

It does not seem to be related to the issue at hand and it is not mentioned in the commit message either.

Comment (KristianB14) [reply themes/ddbasic/sass/components/user.scss]: No - this was a mistake and has been removed in new commit

Comment (kasperg) [reply themes/ddbasic/sass/components/user.scss]: :+1:
